src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/2.diff.sql

```diff
@@ -249,7 +249,7 @@ AS
         
         -- Check if tag exsit
         DECLARE @tagStatus TINYINT
-        DECLARE @tagKey TINYINT
+        DECLARE @tagKey BIGINT
```

Comment:
Can you add a little bit more reasoning on why the change in the description. Will we ever support 8bytes, 2power63 different custom tags.

Reply:
@tagKey is a variable to save tagKey, so that it should be as same type as TagKey. this is a typo.